tfx/benchmarks/benchmark_base.py

```diff
@@ -27,14 +28,73 @@
     "Beam runner to use - any runner name accepted by "
     "apache_beam.runners.create_runner")
 
+modes = ["default", "local_scaled_execution", "cloud_dataflow", "flink_on_k8s"]
+beam_pipeline_mode = "local_execution"
+num_workers_str = "1"
+cloud_dataflow_temp_loc = ""
+
+def set_beam_pipeline_mode(mode):
+  assert mode in modes
+  globals()['beam_pipeline_mode'] = mode
+
+def set_num_workers(num_workers):
+  globals()['num_workers_str'] = str(num_workers)
+
+def set_cloud_dataflow_temp_loc(temp_loc):
+  globals()['cloud_dataflow_temp_loc'] = temp_loc
 
 class BenchmarkBase(test.Benchmark):
+  """Base class for running Beam pipelines on various runners"""
 
-  def _create_beam_pipeline(self):
+  def _set_cloud_dataflow_flags(self):
+    self.flags = ['--runner=DataflowRunner',
```

Comment:
nit: in general we prefer the same type (single or double) of quotes for all strings within a file, even though [PEP-8](https://legacy.python.org/dev/peps/pep-0008/#string-quotes) doesn't make an explicit recommendation about it.

Reply:
Understood, I was always curious about double vs single quotes haha. I switched everything to double quotes.